README.md

```diff
@@ -87,12 +87,11 @@ There is an official image, built from source at: https://hub.docker.com/r/nofis
 
 ### Install from source
 
- - `wget https://github.com/HelloZeroNet/ZeroNet/archive/py3/ZeroNet-py3.tar.gz`
- - `tar xvpfz ZeroNet-py3.tar.gz`
- - `cd ZeroNet-py3`
- - `sudo apt-get update`
- - `sudo apt-get install python3-pip`
- - `sudo python3 -m pip install -r requirements.txt`
+ - `sudo su`
+ - `apt update && apt full-upgrade && apt install git python3-pip`
+ - `git clone https://github.com/hellozeronet/zeronet`
```

Comment:
`git` is not available on many machines by default, why didn't you like the way ZeroNet is downloaded now?